hack/prepare-helmchart.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x 
+
+# ${1} is the given application-name from make (csi-external-snapshotter-app)
+
+# as we apply the crd via configmap, name must be stripped to be RFC 1123 conform
+# (lower case alphanumeric characters or '-', and must start and end with an alphanumeric character)
```

Comment:
Do we apply them via configmap?

Reply:
yes. with crd-install job.

Reply:
Where is this job defined?

Reply:
It is already a part of the main branch now. This PR just adds a webhook part and uses kustomization to generate manifests.  https://github.com/giantswarm/csi-external-snapshotter-app/tree/main/helm/csi-external-snapshotter-app/templates/crd-install